sb-update

```diff
@@ -31,12 +31,22 @@ def argument_parser():
 
 def rebuild_apt_repo():
     logging.debug("Rebuilding apt repo")
-    working_dir = os.getcwd()
-    os.chdir(SB_DEBS)
-    with open('Packages', 'wb') as packages:
-        subprocess.check_call(['dpkg-scanpackages', '.', '/dev/null'], stdout=packages)
-    subprocess.check_call(['xz', '-f', '-3', 'Packages'])
-    os.chdir(working_dir)
+
+    with (SB_DEBS_PATH / 'Packages.xz').open('wb') as packages_xz:
+        scanpackages_process = subprocess.Popen(
+            ['dpkg-scanpackages', '.', '/dev/null'],
+            cwd=SB_DEBS,
+            stdout=subprocess.PIPE,
+        )
+        subprocess.check_call(
+            ['xz', '-3', '-'],
```

Comment:
Python appears to have [standard library support](https://docs.python.org/3/library/lzma.html) for xz, so compressing the Packages file could be done within the application, which saves spawning a child process. My minimal knowledge of Debian packaging suggests the file is probably so small that it's not worth worrying about this, but I could be wrong.

Reply:
The chances of file size being an issue on this scale is very small. However, I think using libraries wherever possible is preferable to calling out to shell